book/6-computer-vision.tex

```diff
@@ -5,4 +5,71 @@ \chapter{Computer Vision}
 \clearpage
 \thispagestyle{cvstyle}
 \section{Computer Vision Sample Metric}
-\subsection{Computer Vision Sample Metric}
\ No newline at end of file
+\subsection{Computer Vision Sample Metric}
+
+
+% ---------- PSNR ----------
+\clearpage
+\thispagestyle{cvstyle}
+\section{PSNR}
+\subsection{Peak Signal-to-Noise Ratio}
+
+In image generation and reconstruction, it is important to measure how closely generated images resemble reference images. 
+PSNR is a commonly used metric for comparing pixel differences between them.
+
+% equation
+\begin{center}
+    \tikz{
+        \node[inner sep=2pt, font=\Large] (a) {
+            {
+                $\displaystyle
+                    \text{PSNR} = 10 \cdot \log_{10}\left(\frac{\color{nmlred}MAX^2}{\color{nmlpurple}MSE}\right)
+                $
+            }
+        };
+        \draw[-latex,nmlred, semithick] ($(a.north)+(2,-0.1)$) to[bend left=15] node[pos=1, left] {maximum possible pixel} +(-1,.5); 
+        \draw[-latex,nmlpurple, semithick] ($(a.south)+(2,-0.1)$) to[bend left=15] node[pos=1, left] {mean squared error} +(-1,-.5);
+    }
+\end{center}
+
+\vspace{-10pt} 
+
+and 
```

Comment:
I'm not a huge fan of having two big equations on a single page, but I can see how, in this case, it is necessary. Just that instead of a lonely "and" I would write something like:

Where given an original image $O$ of size $h \times w$ and a reconstructed image $R$, the $MSE$ is defined as: